apps.json

```diff
@@ -76,5 +76,10 @@
     "name": "Steemit.LOL",
     "homepage": "https://steemit.lol/",
     "url_scheme": "https://steemit.lol/{category}/@{username}/{permlink}"
+  },
+  "steemia": {
+    "name": "Steemia",
+    "homepage": "https://steemia.io/",
+    "url_scheme": "https://steemia.io/{category}/@{username}/{permlink}"
```

Comment:
Link to post are not working for me https://steemia.io/test/@siol/5ubafu-ho-yeah , if you don't have a page display post on your website you should remove the param `url_scheme`